drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/analysis/AnalyzerProvider.java

```diff
@@ -68,20 +79,67 @@ public Status getStatus() {
     }
 
     public DecisionTableAnalyzer getAnalyser( final GuidedDecisionTable52 table52 ) {
-        return new DecisionTableAnalyzerMock( oracle,
-                                              table52,
-                                              new Callback<AnalysisReport>() {
-                                                  @Override
-                                                  public void callback( final AnalysisReport result ) {
-                                                      analysisReport = result;
-                                                  }
-                                              },
-                                              new Callback<Status>() {
-                                                  @Override
-                                                  public void callback( final Status result ) {
-                                                      status = result;
-                                                  }
-                                              } );
+        if ( checkRunner == null ) {
+            //loop
+            checkRunner = new CheckRunner() {
+                @Override
+                protected void doRun( final CancellableRepeatingCommand command ) {
+                    while ( command.execute() ) {
+                        //loop
+                    }
+                }
+            };
+        }
+
+        final UpdateManager updateManager = getUpdateManager( checkRunner,
+                                                              table52 );
+
+        final DecisionTableAnalyzer analyzer = new DecisionTableAnalyzer(
+                new AnalysisReporter( mock( PlaceRequest.class ),
+                                      mock( AnalysisReportScreen.class ) ) {
+                    @Override
+                    public void sendReport( final AnalysisReport report ) {
+                        analysisReport = report;
+                    }
+
+                    @Override
+                    public void sendStatus( final Status _status ) {
+                        status = _status;
+                    }
+                },
+                table52,
+                cache,
+                updateManager,
+                checkRunner ) {
```

Comment:
I think I removed and then moved. So this is old code where I removed the block content. Removed the block.

Tried and looks like I start on a new line because of formatting, if I start right after ( then `sendReport` and `sendStatus` methods stay where they are, but the table, cache and so on move to right. The code is not inline.